reposerver/src/main/resources/db/migration/V9__filename_comments.sql

```diff
@@ -0,0 +1,7 @@
+create table `filename_comments` (
```

Comment:
Wasn't in the ticket.

Reply:
So you only do things on the tickets?

Reply:
I prefer requirements in tickets and not in code reviews, yes.

Reply:
I am not adding a requirement.

Reply:
Well you are saying it should be called metadata now because we might add more in the future. Adding more metadata in the future was not part of this task, I didn't even know about it.